frontend/projects/upgrade/src/app/features/dashboard/segments/modals/upsert-private-segment-list-modal/upsert-private-segment-list-modal.component.html

```diff
@@ -22,21 +22,41 @@
     </mat-form-field>
     <mat-form-field *ngIf="selectedListType === LIST_TYPES.SEGMENT" appearance="outline">
       <mat-label class="ft-14-400">Segment</mat-label>
-      <input type="text" matInput formControlName="segment" [matAutocomplete]="auto" class="ft-14-400" />
-      <mat-hint class="form-hint ft-12-400">{{
-        'feature-flags.upsert-private-segment-list-modal' | translate
-      }}</mat-hint>
+      <input
+        type="text"
+        matInput
+        formControlName="segment"
+        [matAutocomplete]="auto"
+        [placeholder]="'feature-flags.upsert-list-modal.segment-placeholder.text' | translate"
+        class="ft-14-400"
+      />
+      <mat-error class="ft-12-400">
+        {{ 'feature-flags.upsert-list-modal.segment.error.message.text' | translate }}
+      </mat-error>
       <mat-autocomplete #auto="matAutocomplete" [displayWith]="displayAsSegmentName" requireSelection="true">
         <mat-option *ngFor="let segment of segmentsOptions$ | async" [value]="segment" class="ft-14-400">
           {{ segment.name }}
         </mat-option>
       </mat-autocomplete>
     </mat-form-field>
     <ng-container *ngIf="selectedListType && selectedListType !== LIST_TYPES.SEGMENT">
-      <app-common-tags-input formControlName="values"></app-common-tags-input>
+      <app-common-tags-input
+        formControlName="values"
+        [inputType]="CommonTagInputType.VALUES"
+        [forceValidation]="forceValidation"
+        (downloadRequested)="onDownloadRequested($event)"
+      ></app-common-tags-input>
       <mat-form-field appearance="outline">
         <mat-label class="ft-14-400">Name</mat-label>
-        <input matInput formControlName="name" class="ft-14-400" />
+        <input matInput formControlName="name" placeholder="e.g., My schools" class="ft-14-400" />
+        <mat-hint class="form-hint ft-12-400">
+          {{
+            (config.title === 'Add Include List'
```

Comment:
translation is definitely something that I am not an expert on good practices for, but I think what we should be doing probably is passing these keys directly from the common-config, so extend that config type to include a 'hint' property (or whatever you want to name it), send in `hint: 'feature-flags.upsert-include-list-modal.name-hint.text'` and then we can do `{{ config.hint | translate }}` without needing this template logic.